dist/index.d.ts

```diff
@@ -497,15 +500,45 @@ interface GPUImageCopyBuffer
   buffer: GPUBuffer;
 }
 
+/** @deprecated */
 interface GPUImageCopyImageBitmap {
   imageBitmap: ImageBitmap;
   origin?: GPUOrigin2D;
```

Comment:
Just to try to trigger deprecation warnings in as many cases as possible:
```suggestion
  /** @deprecated */
  imageBitmap: ImageBitmap;
  /** @deprecated */
  origin?: GPUOrigin2D;
```